cli/src/utils/downloadBin.ts

```diff
@@ -103,18 +61,16 @@ export async function downloadFileIfNotExists(
   }
 
   const system = getSystem();
```

Comment:
is there a way for us to write tests for getSystem()?

Reply:
It's gonna be a bit tricky, because, well, the output depends on system and CPU architecture. :sweat_smile: 

So you can't really assert the output reliably.

Reply:
The only way of asserting it that I can imagine, would be:

```typescript
const system = getSystem();
if os.platform() === "linux && os.arch === "x64") {
    assert(system === "linux-amd64");
}
[...]
```

Which kinda defeats the purpose of this test, since it would just replicate the logic of `getSystem()`.

I think that to ensure it works properly, we should rather come up with some integration tests, which build the CLI and try to create a new project. We could run those tests on Github Actions both for Linux and Mac runners, to ensure that we support both systems.

Reply:
ok sounds good